src/libs/image.c

```diff
@@ -211,6 +212,9 @@ static void _update(dt_lib_module_t *self)
   gtk_widget_set_sensitive(GTK_WIDGET(d->clear_metadata_button), act_on_cnt > 0);
 
   gtk_widget_set_sensitive(GTK_WIDGET(d->refresh_button), act_on_cnt > 0);
+
+  gtk_widget_set_sensitive(GTK_WIDGET(d->set_monochrome_button), act_on_cnt > 0);
```

Comment:
I don't get it right now. What is too slow? Exif reading is of course slow but that keeps the bit in flags.

You mean for act_on_cnt == 1 it should only the "changing button" should be highlighted?

Reply:
If you have 1000 images selected, checking the flags for each of them is too costly/slow. My proposal is then:
- if multiple images are selected we always have both buttons active
- if a single image is selected, we do check the current flags to enable only one of them

Sorry for not being clear first time, and hoping this makes everything clear now :)

Reply:
I think i get it, you mean it would be too costly to check all selected images just for the `gtk_widget_set_sensitive`. Sure :+1: Will do tomorrow ...